src/NuGet.Services.BasicSearch.Cloud/NuGet.Services.BasicSearch.Cloud.ccproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>2.7</ProductVersion>
+    <ProductVersion>2.9</ProductVersion>
```

Comment:
I can roll this back, or we can move to SDK 2.9 properly. BTW 2.7 is no longer available to install by default